activesupport/lib/active_support/callbacks.rb

```diff
@@ -736,8 +736,13 @@ def reset_callbacks(name)
       #
       #   would call <tt>Audit#save</tt>.
       #
-      # NOTE: +method_name+ passed to `define_model_callbacks` must not end with
+      # ===== Notes
```

Comment:
yes, https://github.com/rails/rails/blob/master/actionpack/lib/action_controller/metal/http_authentication.rb#L161

I just added 2 more `=` to be consistent with the `===== Option` [above](https://github.com/rails/rails/blob/master/activesupport/lib/active_support/callbacks.rb#L599)